### PR TITLE
feat: deprecated logLevel option

### PR DESCRIPTION
**Pull request by viceice**
## Changes:
Deprecate `logLevel` config option.

## Context:
#8291 

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added unit tests, or
- [x] Unit tests + ~ran on a real repository~

### CODE REVIEW

File: docs/development/configuration.md [hunk 1]
```diff
@@ -53,6 +52,12 @@ $ node renovate --help
 To configure any `<list>` items, separate with commas.
 E.g. `renovate --labels=renovate,dependency`.
 
+To enable debug logging run:
+
+```sh
+$ LOG_LEVEL=debug node renovate
```
Comment (rarkins) [inline docs/development/configuration.md:58]: node ?

Comment (viceice) [reply docs/development/configuration.md:58]: It's a copy of the command above. mabe we should use `yarn start` instead 🙃

**Review by rarkins — APPROVED** [review]

**Comment by rarkins** [comment]
@viceice I discovered that this warns even if you *do* define LOG_LEVEL=debug in env only, so I have reverted it. Do you want to create a new PR out of the restored branch? I think we need to check specifically in file and CLI config only.

**Comment by viceice** [comment]
Sure, I'll check now why it warns

**Comment by viceice** [comment]
ha, forgot to update this:
https://github.com/renovatebot/renovate/blob/6e4ae8b9f623f41a80be0e35fe6457e3995c5eb9/lib/config/definitions.ts#L349

🤦‍♂️

**Pull request merged**